tests/csl-json/excess-date-parts-csl/raw.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
What does `excess-date-parts` refer to in this test case?  I'm not sure what is being tested.

Reply:
Fixed in https://github.com/greenelab/manubot/pull/65/commits/4428dadc07b310861d943b63be9a59d45f62d2cf

Reply:
Fixed in https://github.com/greenelab/manubot/pull/65/commits/dfda71cb9ee0e8bffa7bae09e07e6791c7c14707